charts/pomerium/README.md

```diff
@@ -135,6 +156,8 @@ A full listing of Pomerium's configuration variables can be found on the [config
 | `proxy.existingTLSSecret`             | Name of existing TLS Secret for proxy service                                                                                                                                                                                                                                                      |                                                                                       |
 | `proxy.deployment.annotations`        | Annotations for the proxy deployment. If none given, then use value of `annotations`                                                                                                                                                                                                               | `{}`                                                                                  |
 | `proxy.service.annotations`           | Annotations for the proxy service. If none given, then use value of `service.annotations`                                                                                                                                                                                                          | `{}`                                                                                  |
+| `proxy.existingSigningKeySecret`      | Name of existing Signing key Secret for proxy requests.                                                                                                                                                                                                                              |                                                                                       |
+| `proxy.signingKey`                    | Signing key is the base64 encoded key used to sign outbound requests.                                                                                                                                                                                                                              |                                                                                       |
```

Comment:
@tgomas is this still used?

Reply:
Yes, like for tls secrets it is still possible to directly set the value

Reply:
ok, I see there was a missing file when I made the comment :)